spec/dependencies.spec.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+describe("dependencies", () => {
+    var dependencies;
+
+    beforeEach(() => {
+        dependencies = require("../lib/dependencies");
+    });
+
+    it("resolves a dependency", () => {
+        expect(dependencies).toEqual(jasmine.any(Object));
```

Comment:
This line doesn't make sense when you say you are testing if "it resolves a dependency".  I suggest moving it into its own test.